### PR TITLE
Ps 5 fix lavoz scrapper

### PR DESCRIPTION
**Pull request by NicoBellanich**
**La Voz Scrapping**
-id scrapping & house scrapping already done. It gets the phone and all the metadata
-microservice to get phone, disabled

**Olx scrapping disabled**

### CODE REVIEW

**Comment by santimaldonado2** [comment]
Also si a good practice to format the code before committing. The shortcut in PyCharm is "Ctrl+Alt+Shift+L"

File: src/lavozScrapper.py [hunk 1]
```diff
@@ -52,6 +52,8 @@ def get_house_info(self, link):
         info = {}
         if response_house:
 
+            # metas = response_house.find_all("meta") es para ver los metadatos
```
[the remark this answers is not in the document]
Comment (NicoBellanich) [reply src/lavozScrapper.py]: done

File: src/lavozScrapper.py [hunk 2]
```diff
@@ -70,6 +72,8 @@ def get_house_info(self, link):
                 p = re.compile(r'/\d+/')
                 art_id = p.search(link).group(0)[1:-1]
 
+            # microservico - DELETE
```
[the remark this answers is not in the document]
Comment (NicoBellanich) [reply src/lavozScrapper.py]: done

File: src/lavozScrapper.py [hunk 3]
```diff
@@ -104,28 +112,31 @@ def scrap_ids(self):
...
-                page_part = "page=" + str(i) + "&"
+            for i in range(self.from_page, self.from_page + self.pages - 1):
+                page_part = "&page=" + str(i)
 
                 if i == 0:
                     page_part = ""
 
-                search_url_inmu = "https://clasificados.lavoz.com.ar/search/apachesolr_search?" + page_part + "f[0]=im_taxonomy_vid_34%3A6330&f[1]=ss_rol%3A" + publisher_type
+                ##https://clasificados.lavoz.com.ar/buscar/inmuebles?filters%5Bvendedor%5D%5B0%5D=particular&page=1
+                search_url_inmu = "https://clasificados.lavoz.com.ar/buscar/inmuebles?filters%5Bvendedor%5D%5B0%5D=" + publisher_type + page_part
```
[the remark this answers is not in the document]
Comment (NicoBellanich) [reply src/lavozScrapper.py]: done

File: src/lavozScrapper.py [hunk 4]
```diff
@@ -104,28 +112,31 @@ def scrap_ids(self):
...
-            for i in range(self.from_page - 1, self.from_page + self.pages - 1):
-                page_part = "page=" + str(i) + "&"
+            for i in range(self.from_page, self.from_page + self.pages - 1):
+                page_part = "&page=" + str(i)
 
                 if i == 0:
                     page_part = ""
 
-                search_url_inmu = "https://clasificados.lavoz.com.ar/search/apachesolr_search?" + page_part + "f[0]=im_taxonomy_vid_34%3A6330&f[1]=ss_rol%3A" + publisher_type
+                ##https://clasificados.lavoz.com.ar/buscar/inmuebles?filters%5Bvendedor%5D%5B0%5D=particular&page=1
```
[the remark this answers is not in the document]
Comment (NicoBellanich) [reply src/lavozScrapper.py]: done

File: src/lavozScrapper.py [hunk 5]
```diff
@@ -104,28 +112,31 @@ def scrap_ids(self):
...
                 if (not house_list) or len(house_list) <= 0:
                     logger.info(
                         "Error trying to get the houses list: number[{}] publisher_type[{}] url[{}]".format(i + 1,
                                                                                                             publisher_type,
                                                                                                             search_url_inmu))
                     continue
-                house_items = house_list[0].find_all("div", {"class": "BoxResultado"})
-                house_urls = [house_item.div.div.a.get('href') for idx, house_item in enumerate(house_items)]
+
+                ##house_items = house_list[0].find_all("div", {"class": "text-decoration-none"})
```
[the remark this answers is not in the document]
Comment (NicoBellanich) [reply src/lavozScrapper.py]: done

File: src/lavozScrapper.py [hunk 6]
```diff
@@ -118,8 +97,8 @@ def scrap_ids(self):
                 if i == 0:
                     page_part = ""
 
-                ##https://clasificados.lavoz.com.ar/buscar/inmuebles?filters%5Bvendedor%5D%5B0%5D=particular&page=1
-                search_url_inmu = "https://clasificados.lavoz.com.ar/buscar/inmuebles?filters%5Bvendedor%5D%5B0%5D=" + publisher_type + page_part
+                search_url_inmu = 'https://clasificados.lavoz.com.ar/buscar/inmuebles?filters={{"vendedor":{publisher_types}}}{page_part}'.format(
```
Comment (santimaldonado2) [inline src/lavozScrapper.py]: this is wrong. You will be scrapping for each publisher types all publisher types.... 
Opcion 1) Remove the for and do it once
Opcion 2) Replace {{"vendedor":{publisher_types}}} -> {{"vendedor":[{publisher_type}]}}

In the option 1 you will scrap 2 pages for each publisher type, in the option 2, you will scrap 2 pages including all the publisher types.

**Pull request merged**